apps/backoffice/mocks/handlers/services-cms-handlers.ts

```diff
@@ -64,126 +65,126 @@ export const buildHandlers = () => {
           ctx.status(200),
           ctx.json(getUpdateService200Response(serviceId as string))
         ],
-        [ctx.status(400), ctx.json(null)],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(400), ctx.json(getIoServicesError(400))],
+        [ctx.status(401), ctx.json(getIoServicesError(401))],
+        [ctx.status(403), ctx.json(getIoServicesError(403))],
+        [ctx.status(404), ctx.json(getIoServicesError(404))],
+        [ctx.status(429), ctx.json(getIoServicesError(429))],
+        [ctx.status(500), ctx.json(getIoServicesError(500))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.delete(`${baseURL}/services/:serviceId`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(204)],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(401), ctx.json(getIoServicesError(401))],
+        [ctx.status(403), ctx.json(getIoServicesError(403))],
+        [ctx.status(404), ctx.json(getIoServicesError(404))],
+        [ctx.status(429), ctx.json(getIoServicesError(429))],
+        [ctx.status(500), ctx.json(getIoServicesError(500))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.put(`${baseURL}/services/:serviceId/logo`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(204)],
-        [ctx.status(400), ctx.json(null)],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(400), ctx.json(getIoServicesError(400))],
+        [ctx.status(401), ctx.json(getIoServicesError(401))],
+        [ctx.status(403), ctx.json(getIoServicesError(403))],
+        [ctx.status(404), ctx.json(getIoServicesError(404))],
+        [ctx.status(429), ctx.json(getIoServicesError(429))],
+        [ctx.status(500), ctx.json(getIoServicesError(500))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.get(`${baseURL}/services/:serviceId/keys`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(200), ctx.json(getGetServiceKeys200Response())],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(401), ctx.json(getIoServicesError(400))],
```

Comment:
why do we sometimes map each error code to the equivalent one, and in some other cases we just return 400 ?